File: app/src/main/java/android/mb00/codehub/api/model/Repo.java
package android.mb00.codehub.api.model;

import com.google.gson.annotations.SerializedName;


public class Repo {

    @SerializedName("name")
    private String name;
    @SerializedName("full_name")
    private String fullName;
    @SerializedName("description")
    private String description;
    @SerializedName("language")
    private String language;
    @SerializedName("stargazers_count")
    private String stargazersCount;
    @SerializedName("forks_count")
    private String forksCount;
    @SerializedName("owner")
    private Owner owner;

    public String getName() {
        return name;
    }

    public String getFullName() {
        return fullName;
    }

    public String getDescription() {
        return description;
    }

    public String getLanguage() {
        return language;
    }

    public String getStargazersCount() {
        return stargazersCount;
    }

    public String getForksCount() {
        return forksCount;
    }

    public Owner getOwner() {
        return owner;
    }

}

File: app/src/main/java/mb00/android/codehub/api/model/Repo.kt
package mb00.android.codehub.api.model

import com.google.gson.annotations.SerializedName


data class Repo(
        @SerializedName("name") var name: String?,
        @SerializedName("full_name") var fullName: String?,
        @SerializedName("description") var description: String?,
        @SerializedName("language") var language: String?,
        @SerializedName("stargazers_count") var stargazersCount: String?,
        @SerializedName("forks_count") var forksCount: String?,
        @SerializedName("owner") var owner: Owner?
)
